### PR TITLE
feat: Adding Redpanda Blueprint to Streaming Module

### PR DESCRIPTION
**Pull request by ashapiaws**
### What does this PR do?

🛑 Please open an issue first to discuss any significant work and flesh out details/direction - we would hate for your time to be wasted.
Consult the [CONTRIBUTING](https://github.com/awslabs/data-on-eks/blob/main/CONTRIBUTING.md#contributing-via-pull-requests) guide for submitting pull-requests.

This is a PR to add my work on [Redpanda](https://redpanda.com/?utm_source=google&utm_medium=cpc&utm_campaign=redpanda-branded-search&utm_term=redpanda&hsa_acc=3207992515&hsa_cam=17923500714&hsa_grp=138919893599&hsa_ad=687119041570&hsa_src=g&hsa_tgt=kwd-301810926430&hsa_kw=redpanda&hsa_mt=e&hsa_net=adwords&hsa_ver=3&gad_source=1&gclid=Cj0KCQjwwMqvBhCtARIsAIXsZpYIHzV55Qh8Ng-iAPqRGh9qQMhwgEwccck736uYX0TxahO32zCY3EkaAnwkEALw_wcB) Blueprint to Streaming options.

### Motivation

After working internally on this would like to contribute it to dataoneks overall blueprints, Redpanda is streaming alternative to Kafka that can help customers lower cost and reap performance benefits. 

### More

- [ X] Yes, I have tested the PR using my local account setup (Provide any test evidence report under Additional Notes)
- [ ] Mandatory for new blueprints. Yes, I have added a example to support my blueprint PR
- [ ] Mandatory for new blueprints. Yes, I have updated the `website/docs` or `website/blog` section for this feature
- [ ] Yes, I ran `pre-commit run -a` with this PR. Link for installing [pre-commit](https://pre-commit.com/) locally

### For Moderators

- [ ] E2E Test successfully complete before merge?

### CODE REVIEW

File: streaming/redpanda/main.tf [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    azs = slice(data.aws_availability_zones.available.names, 0 ,2)
+
+
+    tags = {
+        
+    }
+
+}
+
+#---------------------------------------
```
Comment (ovaleanu) [inline streaming/redpanda/main.tf:37]: Please move Redpanda Config, Grafana Admin credentials in addons.tf

File: streaming/redpanda/addons.tf [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+        name  = "grafana.adminPassword"
+        value = data.aws_secretsmanager_secret_version.grafana_password_version.secret_string
+      }
+    ],
+  }
+
+  tags = local.tags
+}
+
+ ## Cert-Manager Config
```
Comment (ovaleanu) [inline streaming/redpanda/addons.tf:157]: Cert manager is part of the eks blueprints addons. You can use `enable_cert_manager=true`.

File: streaming/redpanda/addons.tf [hunk 3]
```diff
@@ -0,0 +1,191 @@
...
+    create_namespace = true
+    set {
+     name = "installCRDs"
+     value = true 
+    }
+    depends_on = [ module.eks_blueprints_addons ]
+  }
+  
+  ## Redpanda Helm Config
+resource "helm_release" "redpanda" {
```
Comment (ovaleanu) [inline streaming/redpanda/addons.tf:173]: Redpanda needs to be added in eks_data_addons, https://github.com/aws-ia/terraform-aws-eks-data-addons

File: streaming/redpanda/templates/redpanda_values.yaml
Comment (ovaleanu) [inline streaming/redpanda/templates/redpanda_values.yaml:1]: Please use dash in file name, redpanda-values.yaml. Folder name it should be named helm-values.

File: streaming/redpanda/README.md [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+```
+aws eks --region us-west-2 update-kubeconfig --name doeks-redpanda
+```
+Verify Cluster
+```
+kubectl get nodes
+```
+You should see 6 worked nodes, 3 deployed for EKS Add-ons (Cert-Manager, AWS Load Balancer Controller, Monitoring Tools, etc).
+
+## Configuring Environment for Lambda/Serverless Lab
```
Comment (ovaleanu) [inline streaming/redpanda/README.md:86]: Please remove Lambda references since is not anymore used.

Comment (curiouspacket) [reply streaming/redpanda/README.md:86]: Complete

File: streaming/redpanda/main.tf (hunk 1 above)
Comment (ashapiaws) [reply streaming/redpanda/main.tf]: complete

File: streaming/redpanda/addons.tf (hunk 2 above)
Comment (ashapiaws) [reply streaming/redpanda/addons.tf]: complete

File: streaming/redpanda/templates/redpanda_values.yaml
Comment (ashapiaws) [reply streaming/redpanda/templates/redpanda_values.yaml:1]: complete

**Comment by askulkarni2** [comment]
We do not plan to add this blueprint at this time. Thank you for your work on the PR!

**Pull request closed without merging**